Source/Math/NcclComm.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2016, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Please also add this line below:
// Licensed under the MIT license. See LICENSE.md file in the project root for full license information.

(Regarding headers for existing files: please rebase one more time onto latest master, and add your additional line; we've added some headers where missing - thanks!)